cilium/cmd/lrp.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017 Authors of Cilium
```

Comment:
```suggestion
// Copyright 2020 Authors of Cilium
```

Reply:
There's a few occurrences of this for new files.

Reply:
Please address this before merging.

Reply:
Don't think these have been addressed

Reply:
Done.